src/TestableIO.System.IO.Abstractions/IDirectory.cs

```diff
@@ -20,17 +18,63 @@ public interface IDirectory
         /// <inheritdoc cref="Directory.CreateDirectory(string,DirectorySecurity)"/>
 #endif
         IDirectoryInfo CreateDirectory(string path, DirectorySecurity directorySecurity);
+
 #if FEATURE_CREATE_SYMBOLIC_LINK
-        /// <inheritdoc cref="Directory.CreateSymbolicLink"/>
+        /// <inheritdoc cref="Directory.CreateSymbolicLink(string, string)" />
         IFileSystemInfo CreateSymbolicLink(string path, string pathToTarget);
 #endif
-        /// <inheritdoc cref="Directory.Delete(string)"/>
+
+        /// <inheritdoc cref="Directory.Delete(string)" />
         void Delete(string path);
-        /// <inheritdoc cref="Directory.Delete(string,bool)"/>
+
+        /// <inheritdoc cref="Directory.Delete(string, bool)" />
         void Delete(string path, bool recursive);
-        /// <inheritdoc cref="Directory.Exists"/>
-        bool Exists(string path);
 
+        /// <inheritdoc cref="Directory.EnumerateDirectories(string)" />
+        IEnumerable<string> EnumerateDirectories(string path);
```

Comment:
I see you did some reordering here, that's cool!

Did you use an automated tool for this? If so, which?

Reply:
I use [ReSharper](https://www.jetbrains.com/resharper/) to keep a consistent file layout and code style.
But due to the many `#if` / `#endif` structures, the alphabetical sorting does not work very good, so I had to do it by hand.

Without it, it would be hard to double-check it against my existing interface implementation :-)

Reply:
Ah, I see. Was just wondering if we should add some more automation to automatically handle this. All good :)